src/transformers/models/__init__.py

```diff
@@ -17,6 +17,7 @@
 # limitations under the License.
 
 from . import (
+    qdqbert,
```

Comment:
This will need to be properly sorted, you should run `make style` on your branch :-)